pbsmetrics/prometheus/prometheus.go

```diff
@@ -153,11 +156,16 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 		"Count of stored request cache requests attempts by hits or miss.",
 		[]string{cacheResultLabel})
 
-	metrics.timeout_notifications = newCounter(cfg, metrics.Registry,
+	metrics.timeoutNotifications = newCounter(cfg, metrics.Registry,
 		"timeout_notification",
 		"Count of timeout notifications triggered, and if they were successfully sent.",
 		[]string{successLabel})
 
+	metrics.tcfMetrics = newCounter(cfg, metrics.Registry,
+		"privacy_tcf",
```

Comment:
Similar naming concern as with Influx. How about 'request_privacy_tcf'?

Reply:
The change to the Influx name looks good. Can you please change the name here to match?

Reply:
I don't need to change the name since I made the source a label instead. If we add more sources (usersync), we will want the same version options, so will be the same cardinality as having per source metrics.

Reply:
Cool. Could you please add this to the preload code?